lib/std/RingBuffer.zig

```diff
@@ -11,14 +11,15 @@
 
 const Allocator = @import("std").mem.Allocator;
 const assert = @import("std").debug.assert;
+const copy = @import("std").mem.copy;
```

Comment:
https://github.com/ziglang/zig/blob/b3aaf854cdd814998a09195ba05f5e2bad4c60cb/lib/std/mem.zig#L195-L197

Reply:
modified to support both `@memcpy` and `copyForwards` via separate methods, no longer using deprecated `copy`